cogs/PlaytesterPing.py

```diff
@@ -15,17 +15,10 @@ def getLatestDiscussion():
     query = """
     query {
         repository(owner: "R2Northstar", name: "Northstar") {
-            discussions(first: 1) {
+            discussions(first: 1 categoryId: "DIC_kwDOGkM8Yc4CN-04") {
                 edges { 
                     node {
-                        author {
-                            login
-                            url
-                        }
-                        title
-                        number
                         body
-                        url
```

Comment:
I do actually think keeping the url would be nice. In the current version ( as in, the playtester ping Gecko does manually ), the discussion is linked to. I'll provide a rough example where I'd like it to be used (you may need to fix the code; all of the api stuff in Spectre wasn't written by me)